include/zephyr/kernel/internal/sched_priq.h

```diff
@@ -10,6 +10,18 @@
 #include <zephyr/sys/dlist.h>
 #include <zephyr/sys/rb.h>
 
+#define K_HIGHEST_THREAD_PRIO             (-CONFIG_NUM_COOP_PRIORITIES)
```

Comment:
This part already defined in <zephyr/kernel.h> but this file can't be included in this file, it will cause compiler error.
Does anyone have suggestions about this code?